helm/trino-gateway/values.yaml

```diff
@@ -110,6 +111,11 @@ tolerations: []
 
 affinity: {}
 
+commonLabels: {}
```

Comment:
Add docs like in https://github.com/trinodb/charts/blob/ac81a3cc14b766d3c2b2ed70e85863f67dc36a2a/charts/trino/values.yaml#L799

That way we can produce docs later like in the Trino chart..

Reply:
added